src/namespaced.ts

```diff
@@ -5,7 +5,9 @@ import {Store} from 'vuex';
 export type Nullish = null | undefined;
 
 function computedState(store: any, namespace: string, prop: string) {
-	return computed(() => store.state[namespace][prop])
+	return computed(() => {
+		return namespace.split('/').reduce((module, key) => module[key], store.state)[prop];
```

Comment:
if we already have the store, namespace, and prop, we should probably write this reducer (or most of it) outside of the computed function, because right now it will run this reduce on every value change.

Reply:
Done.